test/createApp.spec.js

```diff
@@ -16,4 +24,78 @@ describe('createApp', () => {
     expect(app).to.be.instanceof(App);
     expect(app.getOption('name')).to.eql('MyAppName');
   });
+
+
+  it('creates an instance with a validationFunctions option and returns it via getValidationFunctions()', () => {
+    const expectedValidationFunctions = ['expectedValue'];
+    const App = createApp({
+      appId: '123',
+      component: true,
+      name: 'MyAppName',
+      validationFunctions: expectedValidationFunctions,
+    });
+
+    const app = new App();
+
+    expect(app).to.be.instanceof(App);
+    expect(app.getValidationFunctions()).to.eql(expectedValidationFunctions);
+  });
+
+  it('creates a RootApp instance with a validationFunctions option and a ChildApp returns them via getValidationFunctions()', () => {
+    const expectedValidationFunctions = ['expectedValue'];
+    const unExpectedValidationFunctions = ['unExpectedValue'];
+    const RootApp = createApp({
+      appId: '123',
+      component: true,
+      name: 'MyRootAppName',
+      validationFunctions: expectedValidationFunctions,
+    });
+    const rootApp = new RootApp();
+    window.app = rootApp;
+
+    const ChildApp = createApp({
+      appId: '234',
+      component: true,
+      name: 'MyChildAppName',
+      validationFunctions: unExpectedValidationFunctions
+    });
+    const childApp = new ChildApp();
+
+
+    expect(rootApp).to.be.instanceof(RootApp);
```

Comment:
I'd say that only the last two assertions are important here, the other _sanity_ checks, i.e. `instanceof` and `eql`, are doing nothing, but testing the test code itself.  Make it shorter and the readability will be better.